test/e2e/kubernetes/scripts/modprobe-config-validate.sh

```diff
@@ -8,4 +8,16 @@ modprobe -n -v dccp | egrep 'install /bin/true' || exit 1
 modprobe -n -v sctp | egrep 'install /bin/true' || exit 1
 [ -s `lsmod | grep sctp` ] || exit 1
 modprobe -n -v rds | egrep 'install /bin/true' || exit 1
-[ -s `lsmod | grep rds` ] || exit 1
\ No newline at end of file
+[ -s `lsmod | grep rds` ] || exit 1
+modprobe -n -v cramfs | egrep 'install /bin/true' || exit 1
+[ -s `lsmod | grep cramfs` ] || exit 1
+modprobe -n -v freevxfs | egrep 'install /bin/true' || exit 1
+[ -s `lsmod | grep freevxfs` ] || exit 1
+modprobe -n -v jffs2 | egrep 'install /bin/true' || exit 1
+[ -s `lsmod | grep jffs2` ] || exit 1
+modprobe -n -v hfs | egrep 'install /bin/true' || exit 1
+[ -s `lsmod | grep hfs` ] || exit 1
+modprobe -n -v hfsplus | egrep 'install /bin/true' || exit 1
+[ -s `lsmod | grep hfsplus` ] || exit 1
+[ -s `lsmod | grep squashfs` ] || exit 1
```

Comment:
The original request called for explicitly blacklisting "squashfs" and "vfat". As I was implementing this I observed that adding `install squashfs /bin/true` and `install vfat /bin/true` to the modprobe config had no effect, and so I opted to just remove it. I kept in these `lsmod` tests so that we'll know if some other vector introduces them in the future. `lsmod` is the real test here anyway, it'll reveal that any of these modules are present in the active kernel runtime.